### PR TITLE
add topnavbar

### PR DESCRIPTION
**Pull request by shejal00**

### CODE REVIEW

File: portfolio/src/main/webapp/css/navBar.css [hunk 1]
```diff
@@ -0,0 +1,447 @@
+html, body{
```
Comment (abhatia2107) [inline portfolio/src/main/webapp/css/navBar.css:1]: Keep same indentation across the file. Decide if 2 space or 4 space works for you but stick with one.

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -1,18 +1,139 @@
...
+        <section id="projects" class="section">
+            <div class="container">
+                <div class="row">
+                    <div class="col-lg-12">
+                        <h2>Projects</h2>
+                    </div>	
+                </div>
+                <div class="row">		
+                    <div class="col-lg-12" style="padding:0px;">
+                        <div class='col-sm-4' style='margin:0px;padding:1vh;text-align:center;'>
```
Comment (abhatia2107) [inline portfolio/src/main/webapp/index.html:84]: Avoid using inline css. Shift this to style.css

File: portfolio/src/main/webapp/index.html [hunk 3]
```diff
@@ -1,18 +1,139 @@
...
+        </section>
+
+        <footer>
+            <div class="container"></div>
+        </footer>
+        <footer>
+        <hr style="border-width:1px;border-color:#000000;"">
+        <div id="footer">
+            <h4 style="float:left;padding:0px 10px 30px 0px;color:#000000;font-size:x-large"><b>&copy; 2020 Shejal Gupta</b></h4>
+            </div>
```
Comment (abhatia2107) [inline portfolio/src/main/webapp/index.html:131]: nit: closing indentation doesn't look right.

File: portfolio/src/main/webapp/index.html [hunk 4]
```diff
@@ -1,18 +1,139 @@
...
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
-    </div>
-  </body>
+    <head>
+        <meta charset="UTF-8">
+        <title>My Portfolio</title>
+        <link rel="stylesheet" href="style.css">
```
Comment (abhatia2107) [inline portfolio/src/main/webapp/index.html:6]: Should add type="text/css" with all the stylesheets.

File: portfolio/src/main/webapp/index.html [hunk 5]
```diff
@@ -1,18 +1,139 @@
...
+        <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
+        <link rel="stylesheet" href="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js">
```
Comment (abhatia2107) [inline portfolio/src/main/webapp/index.html:8]: js files should be included at the end of file and they shouldn't be added as stylesheet but as script instead. 
https://stackoverflow.com/questions/6625773/where-should-i-put-the-css-and-javascript-code-in-an-html-webpage

**Comment by shejal00** [comment]
All the suggested changes are made/

**Pull request closed without merging**